src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -26,14 +26,14 @@ export type SortColumn = {
   ascending: boolean
 } | null
 
-const DataViewTable: React.FC = () => {
+const DataViewTable: React.FC<{ codeFromLookupTable?: string }> = ({ codeFromLookupTable }) => {
```

Comment:
This is how the "DataViewTable" can tell whether we're loading from a lookup table page or a standard "Data View" table page -- the presence or absence of the "codeFromLookupTable" prop.